css2/bikeshed-header.bs.part

```diff
@@ -0,0 +1,51 @@
+<pre class='metadata'>
+Title: CSS 2
+Shortname: CSS2
+Level:
+Status: ED
+Work Status: Stable
+Group: csswg
+ED: https://drafts.csswg.org/css2/
+TR: https://www.w3.org/TR/CSS2/
+Previous version: https://www.w3.org/TR/2011/REC-CSS2-20110607/
+Previous version: https://www.w3.org/TR/2008/REC-CSS2-20080411/
+Editor: Sam Sneddon, https://gsnedders.com
+Editor: Tantek &Ccedil;elik, tantek@cs.stanford.edu
+Former Editor: Editor: Bert Bos, W3C, mailto:bert@w3.org, w3cid 3343
+Former Editor: Elika J. Etemad / fantasai, Invited Expert, http://fantasai.inkedblade.net/contact, w3cid 35400
+Former Editor: Ian Hickson, ian@hixie.ch
+Former Editor: H&aring;kon Wium Lie, Opera Software, howcome@opera.com
+Abstract: foo
+</pre>
+
+<!-- the following is just copy/pasted from specref, is there a better way to
+     define CSS20 as an alias for CSS2-19980512? -->
```

Comment:
Instead of all those contents, use an `aliasOf` key whose value is the biblio name you want to refer to.